templates/tarsnap.key

```diff
@@ -0,0 +1 @@
+{{ TARSNAP_KEY }}
```

Comment:
This one is needed, tarsnap requires newline at the end of file, and yaml strips variables.